pkg/marker/marker.go

```diff
@@ -120,6 +122,8 @@ func Update(nodeName string, cache cache.Cache) error {
 		}
 	}
 
+	glog.Infof("Update: patch operations: %v", patchOperations)
```

Comment:
This does not seem to be useful as a production "info" log. We already report what is the current set of observed bridges. Is this other log useful for anything extra?

Reply:
I think all the logs I added were a mistake, since the Update() function runs every 1 Min this will just spam the logs.
I completely removed them.